core/src/test/java/org/apache/accumulo/core/spi/balancer/SimpleLoadBalancerTest.java

```diff
@@ -59,16 +59,17 @@ static class FakeTServer {
     TServerStatus getStatus() {
       org.apache.accumulo.core.manager.thrift.TabletServerStatus result =
           new org.apache.accumulo.core.manager.thrift.TabletServerStatus();
-      result.tableMap = new HashMap<>();
+      result.setTableMap(new HashMap<>());
       for (TabletId tabletId : tablets) {
-        TableInfo info = result.tableMap.get(tabletId.getTable().canonical());
+        TableInfo info = result.getTableMap().get(tabletId.getTable().canonical());
         if (info == null) {
-          result.tableMap.put(tabletId.getTable().canonical(), info = new TableInfo());
+          result.putToTableMap(tabletId.getTable().canonical(), info = new TableInfo());
```

Comment:
This is a good place to use computeIfAbsent to construct the empty TableInfo.

Reply:
Had that in a previous revision but removed it while trying to debug something. Will add it back.